portfolio/src/main/webapp/script.js

```diff
@@ -45,26 +45,60 @@ function openConnectModal() {
   modal.style.display = "block";
 }
 
-function closeIntroModal(){
+function openQuestionModal() {
+  const modal = document.getElementById("myQuestionModal");
+  modal.style.display = "block";
+}
+
+function openViewModal() {
+  const modal = document.getElementById("myQuestionViewModal");
+  modal.style.display = "block";
+  fetch('/data').then(response => response.json()).then((Questions) => {
+  const questionContainer = document.getElementById("question-container");
+  questionContainer.innerHTML = '';
+  for (let i = 0; i < Questions.length; i++) {
+    const createElement = document.createElement("li");
+    var createNode = document.createTextNode(Questions[i].name + " asked: " + Questions[i].text);
+    createElement.appendChild(createNode);
+    questionContainer.appendChild(createElement);
+    if (i === 9) break;}
```

Comment:
Instead of having a break statement using a raw value at the bottom, I would recommend:

1) defining 9 as a constant. having random raw values scattered throughout your code is bad practice because it makes debugging and reading the code harder.
define `const MAX_QUESTIONS = 9;` at the top of your javascript file
and use it in your break statement like
`if (i === MAX_QUESTIONS) {`
`break;`
`}`

2) move the break statement to the top of your for-loop to make it more readable and clear. 
https://softwareengineering.stackexchange.com/questions/58237/are-break-and-continue-bad-programming-practices